src/lib/components/table/terra-table-data-source.ts

```diff
@@ -131,6 +133,11 @@ export abstract class TerraTableDataSource<T> extends DataSource<T> {
         /* no-op */
     }
 
+    /** Initiates a request that reloads the data with the currently set filters and page data **/
```

Comment:
Kannst du die Methode hoch zur search Methode schieben!? Dann sind die beieinander.
Gut wäre auch noch eine Anmerkung in der JSDoc zur search methode, dass diese nun immer die erste Seite lädt, falls die Daten paginated abgerufen werden.